tsconfig.json

```diff
@@ -13,13 +13,13 @@
     "module": "esnext",
     "lib": ["es2018", "dom"],
     "paths": {
-      "fibonacci": ["dist/fibonacci/fibonacci", "dist/fibonacci"],
+      "fibonacci": ["projects/fibonacci/src/public-api", "dist/fibonacci"],
```

Comment:
Have you just checked that the app can still be built? So that the demo page will be able to deploy? :)

Because I think I tried that at first at worked for TS and IDE but wasn't working for compilation. Unsure now maybe I don't recall correctly or things have changed

Reply:
Well without those changes TS in my IDE cannot find the modules, as mentioned I got 

```
Cannot find module 'fibonacci-webworker' or its corresponding type declarations.ts(2307)
```

It's building on my localhost.

```
> demo-webworker-library@0.0.0 build
> ng build

✔ Browser application bundle generation complete.
✔ Copying assets complete.
✔ Index html generation complete.

Initial Chunk Files | Names                        |  Raw Size | Estimated Transfer Size
main.js             | main                         | 210.00 kB |                56.43 kB
polyfills.js        | polyfills                    |  45.23 kB |                13.85 kB
runtime.js          | runtime                      |   2.82 kB |                 1.30 kB
styles.css          | styles                       |   0 bytes |                       -

                    | Initial Total                | 258.05 kB |                71.59 kB

Lazy Chunk Files    | Names                        |  Raw Size | Estimated Transfer Size
917.js              | demo-lib-consuming-webworker |   3.57 kB |                 1.48 kB
fibonacci.worker.js | fibonacci.worker             | 114 bytes |                89 bytes

Build at: 2022-09-24T18:55:24.741Z - Hash: f7782d94f0137aa9 - Time: 4514ms
```